src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -19,11 +20,13 @@ import TaskDueDate from './TaskDueDate';
 import Status from './TaskStatus';
 import './Task.scss';
 
-type Props = {
+type getAvatarUrlCb = string => Promise<?string>;
```

Comment:
Is this type needed? If it is, for consistency, could this be named `getAvatarUrlCallback`? And should there be a type defined for `getUserProfileUrl` as well then?